src/pybind/cephfs/cephfs.pyx

```diff
@@ -23,6 +23,19 @@ else:
 
 cdef int AT_SYMLINK_NOFOLLOW = 0x100
 cdef int CEPH_STATX_BASIC_STATS = 0x7ffu
+cdef int CEPH_STATX_MODE = 0x1u
+cdef int CEPH_STATX_NLINK = 0x2u
+cdef int CEPH_STATX_UID = 0x4u
+cdef int CEPH_STATX_GID = 0x8u
+cdef int CEPH_STATX_RDEV = 0x10u
+cdef int CEPH_STATX_ATIME = 0x20u
+cdef int CEPH_STATX_MTIME = 0x40u
+cdef int CEPH_STATX_CTIME = 0x80u
+cdef int CEPH_STATX_INO = 0x100u
+cdef int CEPH_STATX_SIZE = 0x200u
+cdef int CEPH_STATX_BLOCKS = 0x400u
+cdef int CEPH_STATX_BTIME = 0x800u
+cdef int CEPH_STATX_VERSION = 0x1000u
```

Comment:
Is there some way to automatically generate these from the header file? Otherwise, if we add new attributes later, then we'll have to update this as well (not a big deal, but it would be nice to have these automatically kept in sync).

Reply:
I haven't thought of a good way, do you have any suggestions?

Reply:
No, I don't. I think we're probably stuck with just duplicating them.

Reply:
we could generate/update `cephfs.pyx` at build time using cmake using a script which reads the cephfs header file. but that'd probably be overkill, i think.

Reply:
thanks for the trick